components/constellation/browser.rs

```diff
@@ -0,0 +1,262 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use std::collections::{HashMap, HashSet};
+
+use msg::constellation_msg::TopLevelBrowsingContextId;
+
+#[derive(Debug)]
+pub struct BrowserManager<Browser> {
```

Comment:
The "Browser" terminology is genuinely confusing! Can we switch to something like "WebView" which is much clearer about what these objects are? Obviously there's a lot of code that uses "Browser," but we can fix that iteratively.

Reply:
An alternative here is ServoView if we want to match the GeckoView terminology. I'm not super attached to any name, as much as opposed to 'Browser' due to the overloaded meaning.

Reply:
done in 4aea74fae2e6ae328104ce45fc63336941366cd7